### PR TITLE
Do not use styles from goodstrap

### PR DESCRIPTION
**Pull request by NestorEncinas**
- use sdk-ui-kit
 - add foundation-sites to sdk-ui lib

JIRA: FET-678

---

Supported PR commands:

| Command                  | Description            |
| ------------------------ | ---------------------- |
| `ok to test`             | Re-run standard checks |
| `extended test`          | BackstopJS tests       |
| `extended check sonar`   | SonarQube tests        |
| `extended check cypress` | Cypress E2E tests      |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `check-extended-cypress` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/5435/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/1173/ : SUCCESS in 17m 36s

**Comment by NestorEncinas** [comment]
extended check sonar

**Comment by NestorEncinas** [comment]
extended check cypress

File: libs/sdk-ui-kit/styles/scss/variables.scss [hunk 1]
```diff
@@ -69,6 +69,9 @@ $gd-palette-success-dimmed: var(
 // secondary colors
 $gd-color-positive-dimmed: mix($gd-color-positive, $default-gd-color-text-light, 10%);
 
+// moved from goodstrap, this variable is used on KD
+$gd-color-negative-dimmed: mix($gd-color-negative, $default-gd-color-text-light, 10%);
```
Comment (kandl) [inline libs/sdk-ui-kit/styles/scss/variables.scss:73]: Should we rather move it just to KD?

**Review by kandl — COMMENTED** [review]
Are there still some dependencies on goodstrap in relevant libs? Or should we remove it also from package.json files (maybe only somewhere)?

File: libs/sdk-ui-kit/styles/scss/variables.scss (hunk 1 above)
Comment (kandl) [reply libs/sdk-ui-kit/styles/scss/variables.scss:73]: And is it used in SDK at all? If not so, maybe we should specify it just in KD? (there are also other KD-only variables, or not?)

Comment (NestorEncinas) [reply libs/sdk-ui-kit/styles/scss/variables.scss:73]: They are not used on SDK, will move them to KD.

**Comment by NestorEncinas** [comment]
extended test

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/1174/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Pull request merged**